index.js

```diff
@@ -205,3 +206,203 @@ export function checkDataIntegrityProofFormat({
+          // FIXME: Fix test to check if a cryptographic suite requires the
+          // “proof.created” value
+          const credential = credentials.clone('invalidCreated');
+          await verificationFail({credential, verifier});
+        });
+        it('If the "proof.created" field is set and it deviates more than ' +
+          '"options.acceptableCreatedTimeDeviationInSeconds" seconds, a ' +
+          '"CREATED_TIME_DEVIATION_ERROR" MUST be raised.', async function() {
+          this.test.cell = {columnId: vendorName, rowId: this.test.title};
+          // FIXME: Fix test to check if a cryptographic suite requires the
+          // “proof.created” value
+          const credential = credentials.clone('issuedVc');
+          // intentionally set the created date to be a week ago
+          const created = new Date();
+          created.setDate(created.getDate() - 7);
+          credential.proof.created =
+            created.toISOString().replace(/\.\d+Z$/, 'Z');
+          await verificationFail({
+            credential, verifier, options: {
+              // set acceptable created time deviation to one day => 86400s
+              acceptableCreatedTimeDeviationInSeconds: 86400
+            }
+          });
+        });
+        it('If the "proof.proofValue" field is not a multibase-encoded ' +
+          'base58-btc value, an "INVALID_PROOF_VALUE" error MUST be raised.',
+        async function() {
+          this.test.cell = {columnId: vendorName, rowId: this.test.title};
+          const credential = credentials.clone('issuedVc');
+          // remove the initial z
+          credential.proof.proofValue = credential.proof.proofValue.slice(1);
+          await verificationFail({credential, verifier});
+        });
+        it('If the "options.domain" is set and it does not match ' +
+          '"proof.domain", an "INVALID_DOMAIN_ERROR" MUST be raised.',
+        async function() {
+          this.test.cell = {columnId: vendorName, rowId: this.test.title};
+          const credential = credentials.clone('issuedVc');
+          // intentionally set the domain to be different from the expected
+          // domain in options
+          credential.proof.domain = 'not-domain.example';
+          await verificationFail({
+            credential, verifier, options: {
+              domain: 'domain.example'
+            }
+          });
+        });
+        it('If the "options.challenge" is set and it does not match ' +
+          '"proof.challenge", an "INVALID_CHALLENGE_ERROR" MUST be raised.',
+        async function() {
+          this.test.cell = {columnId: vendorName, rowId: this.test.title};
+          const credential = credentials.clone('issuedVc');
+          credential.proof.domain = 'domain.example';
+          // intentionally set the challenge to be different from the expected
+          // challenge in options
+          credential.proof.challenge = '1235abcd6789';
```

Comment:
so adding a challenge and a domain to an existing issuedVc will also cause a fail. So we need a real issued VC with a domain and challenge set. Then we need to pass the same domain and a different challenge as options to the verifier using the VC API https://w3c-ccg.github.io/vc-api/#verify-credential

also use example.com here as do know we can use example.com with out issues. In other words you're pretty close, but not quite there.

Reply:
Updated to use VC issued with an invalid challenge [9ba699a](https://github.com/w3c-ccg/data-integrity-test-suite-assertion/pull/13/commits/9ba699a076483b3e155d31cae9559ab780145949)